src/components/AmountEditable.tsx

```diff
@@ -187,13 +171,62 @@ export const AmountEditable: ParentComponent<{
         props.initialAmountSats || "0"
     );
     const [localFiat, setLocalFiat] = createSignal(
-        satsToUsd(
+        satsToFiat(
             state.price,
             parseInt(props.initialAmountSats || "0") || 0,
             false
         )
     );
 
+    const fixedAmount = (multiplier: number, label: boolean) => {
```

Comment:
I'm confused by some of this multiplier logic... the fact that there are hardcoded matches in here makes me think this isn't a very generic function, but I'm scared that the logic will become even harder to follow if it's made generic somehow. maybe we should just discuss the best way to do it

Reply:
Ok! I thought about doing it based on price but you might be right about even more added complexity there

Reply:
I changed the logic to use the price to determine a good fixedAmount to use, should allow for all currenies to have ~$1, ~$10, ~$100 option even if their price is a bit different compared to USD